db/controllers.js/controllers.js

```diff
@@ -58,6 +59,16 @@ exports.updateComments = (req, res, next) => {
     });
 };
 
-exports.updateVotes = (req, res) => {
-  console.log(req);
+exports.updateVotes = (req, res, next) => {
+  const { body } = req;
+  const { article_id } = req.params;
+  console.log(body, article_id);
+  updateArticle(article_id, body)
+    .then((updatedArticle) => {
+      res.status(200).send({ updatedArticle });
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
As above, if you want to make sure what the error is then go ahead and put console logs in your error handlers.